src/test/java/com/shippo/model/BatchTest.java

```diff
@@ -136,6 +136,13 @@ private void waitForBatchStatus(String batchId, BatchStatus status, int times) t
 			if (batch.getStatus().equals(status)) {
 				return;
 			}
+			
+			try {
+				Thread.sleep(1000);
```

Comment:
What is this sleep for?

Reply:
For testing tokens, the server has a 50 request per minute rate limit. Slowing down the rate of calls mitigate that in CircleCI's remote env.